server/cluster.go

```diff
@@ -134,10 +134,12 @@ func (c *RaftCluster) start() error {
 func (c *RaftCluster) runCoordinator() {
 	defer logutil.LogPanic()
 	defer c.wg.Done()
-	defer c.coordinator.wg.Wait()
+	defer func() {
+		c.coordinator.wg.Wait()
+		log.Info("coordinator has been stopped")
+	}()
 	c.coordinator.run()
 	<-c.coordinator.ctx.Done()
-	log.Info("coordinator: Stopped coordinator")
```

Comment:
em, can we keep the old log too?

Reply:
Is this log still useful?

Reply:
I think so. We can tell from the log whether `coordinator.ctx` is cancelled.

Reply:
How about `coordinator is stopping`?

Reply:
LGTM